Source/Core/Core/PowerPC/JitArm64/JitArm64_SystemRegisters.cpp

```diff
@@ -247,38 +247,43 @@ void JitArm64::mfspr(UGeckoInstruction inst)
   case SPR_TL:
   case SPR_TU:
   {
-    // The inline implementation here is inaccurate and out of date as of PR3601
-    FALLBACK_IF(true);  // Fallback to interpreted version.
-
-    /*
     ARM64Reg WA = gpr.GetReg();
     ARM64Reg WB = gpr.GetReg();
     ARM64Reg XA = EncodeRegTo64(WA);
     ARM64Reg XB = EncodeRegTo64(WB);
+    ARM64Reg VC = fpr.GetReg();
+    ARM64Reg VD = fpr.GetReg();
+    ARM64Reg SC = EncodeRegToSingle(VC);
+    ARM64Reg SD = EncodeRegToSingle(VD);
 
     // An inline implementation of CoreTiming::GetFakeTimeBase, since in timer-heavy games the
     // cost of calling out to C for this is actually significant.
-    MOVI2R(XA, (u64)&CoreTiming::g_global_timer);
+    MOVP2R(XA, &CoreTiming::g.last_OC_factor_inverted);
+    m_float_emit.LDR(32, INDEX_UNSIGNED, SC, PPC_REG, PPCSTATE_OFF(downcount));
+    m_float_emit.LDR(32, INDEX_UNSIGNED, SD, XA, 0);
+    m_float_emit.FMUL(SC, SC, SD);
+    m_float_emit.FCVTZS(64, XB, SC);
+    MOVP2R(XA, &CoreTiming::g.slice_length);
     LDR(INDEX_UNSIGNED, XA, XA, 0);
-    MOVI2R(XB, (u64)&CoreTiming::g_fake_TB_start_ticks);
+    SUB(XA, XA, XB);
+    MOVP2R(XB, &CoreTiming::g.global_timer);
+    LDR(INDEX_UNSIGNED, XB, XB, 0);
```

Comment:
as JIT64 doesn't depend on the order, we can use LDP to load global_timer and slice_length at once.